src/lib/data-collector.js

```diff
@@ -28,14 +28,30 @@ DataCollector.prototype.initialize = function () {
     return global.braintree.dataCollector.create(self._config);
   }).then(function (instance) {
     self._instance = instance;
+  }).catch(function (err) {
+    // log the Data Collector setup error
+    // but do not prevent Drop-in from loading
+    self.log(err);
```

Comment:
Do we want analytic events around this?

Reply:
Good idea.

Reply:
Added the event. @jackellenberger and @hollabaq86, are you happy with `data-collector.setup-failed`?